### ISSUE TITLE
Seems to not work when installed only server-side

### ISSUE THREAD

**Issue by NyanMaths** [The-Fireplace-Minecraft-Mods/Unforgiving-Void#20]
### Mod Version

3.1.5

### What mod loader(s) are you seeing the problem on?

Fabric

### Mod Loader Version

0.14.9

### Problem Description

Really simple issue : I only got the mod working by installing it both sides, unless I could not connect to the server, I joined the relevant logs.
Hope it will help !

### Crash Report or Error Logs

```shell
[11:55:45] [Render thread/INFO]: Started 6 worker threads
[11:55:45] [Render thread/INFO]: [bclib] Sending hello to server.
[11:55:45] [Render thread/ERROR]: Registry remapping failed!
net.fabricmc.fabric.impl.registry.sync.RemapException: Received ID map for minecraft:command_argument_type contains IDs unknown to the receiver!
 - minecraft:offline_player
	at net.minecraft.class_2370.remap(class_2370.java:1236) ~[client-intermediary.jar:?]
	at net.fabricmc.fabric.impl.registry.sync.RegistrySyncManager.apply(RegistrySyncManager.java:314) ~[fabric-registry-sync-v0-0.9.24+aeb40ebe90-79c99d818867015a.jar:?]
	at net.fabricmc.fabric.impl.registry.sync.RegistrySyncManager.lambda$receivePacket$0(RegistrySyncManager.java:125) ~[fabric-registry-sync-v0-0.9.24+aeb40ebe90-79c99d818867015a.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) [?:?]
	at net.minecraft.class_1255.method_18859(class_1255.java:157) [client-intermediary.jar:?]
	at net.minecraft.class_4093.method_18859(class_4093.java:23) [client-intermediary.jar:?]
	at net.minecraft.class_1255.method_16075(class_1255.java:131) [client-intermediary.jar:?]
	at net.minecraft.class_1255.method_5383(class_1255.java:116) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1129) [client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:768) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:244) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) [fabric-loader-0.14.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.9.jar:?]
[11:55:45] [REI-ReloadPlugins/INFO]: [REI] Reloading Plugin Manager [REIPlugin] stage [START], registered 10 plugins: DefaultRuntimePlugin [roughlyenoughitems], AlloyForgeryPlugin [alloy_forgery], REIPlugin [betterend], ConjuringClientPlugin [conjuring], ReiCompat [ellemes_container_lib], OwoReiPlugin [owo], DefaultClientRuntimePlugin [roughlyenoughitems], FabricFluidAPISupportPlugin, DefaultPlugin [roughlyenoughitems], DefaultClientPlugin [roughlyenoughitems]
[11:55:45] [Netty Client IO #2/WARN]: Configuration conflict: there is more than one oshi.architecture.properties file on the classpath
[11:55:45] [REI-ReloadPlugins/INFO]: [REI] Reloading Plugin Manager [REIClientPlugin] stage [START], registered 7 plugins: AlloyForgeryPlugin [alloy_forgery], REIPlugin [betterend], ConjuringClientPlugin [conjuring], ReiCompat [ellemes_container_lib], OwoReiPlugin [owo], DefaultClientRuntimePlugin [roughlyenoughitems], DefaultClientPlugin [roughlyenoughitems]
[11:55:45] [REI-ReloadPlugins/INFO]: [REI] Reloading Plugin Manager [REIServerPlugin] stage [START], registered 3 plugins: DefaultRuntimePlugin [roughlyenoughitems], FabricFluidAPISupportPlugin, DefaultPlugin [roughlyenoughitems]
[11:55:45] [REI-ReloadPlugins/INFO]: [REI] Reloaded Plugin Manager [REIPlugin] with 10 plugins in 79.17 ms.
[11:55:45] [REI-ReloadPlugins/INFO]: [REI] Reloaded Plugin Manager [REIClientPlugin] with 7 plugins in 187.4 ms.
[11:55:45] [REI-ReloadPlugins/INFO]: [REI] Reloaded Plugin Manager [REIServerPlugin] with 3 plugins in 1.743 ms.
[11:55:45] [Render thread/INFO]: Stopping worker threads
[11:55:45] [Render thread/INFO]: Started 6 worker threads
[11:55:45] [Render thread/INFO]: Reloading entity icon resources...
[11:55:45] [Render thread/INFO]: Done!
[11:55:45] [Render thread/INFO]: Minimap session finalized.
[11:55:45] [Render thread/INFO]: Stopping worker threads
```

**Comment by The-Fireplace** [comment]
What Minecraft version? 1.19?

**Comment by The-Fireplace** [comment]
Assuming 1.19, this is a duplicate of The-Fireplace-Minecraft-Mods/Fireplace-Lib#51
Leaving this one open for visibility.

**Comment by The-Fireplace** [comment]
Should be fixed with Fireplace Lib 7.0.3
